cmd/osbuild-tests/main_test.go

```diff
@@ -0,0 +1,343 @@
+// +build integration
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/BurntSushi/toml"
+	"github.com/google/uuid"
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
+
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestComposeIntegration(t *testing.T) {
+	// Smoke tests, until functionality tested fully
+	// that the calls succeed and return valid output
+	runComposerCLI(t, false, "compose", "types")
+	runComposerCLI(t, false, "compose", "status")
+	runComposerCLI(t, false, "compose", "list")
+	runComposerCLI(t, false, "compose", "list", "waiting")
+	runComposerCLI(t, false, "compose", "list", "running")
+	runComposerCLI(t, false, "compose", "list", "finished")
+	runComposerCLI(t, false, "compose", "list", "failed")
+	// runCommand(false, "compose", "log", UUID, [<SIZE>])
+	// runCommand(false, "compose", "cancel", UUID)
+	// runCommand(false, "compose", "delete", UUID)
+	// runCommand(false, "compose", "info", UUID)
+	// runCommand(false, "compose", "metadata", UUID)
+	// runCommand(false, "compose", "logs", UUID)
+	// runCommand(false, "compose", "results", UUID)
+	// runCommand(false, "compose", "image", UUID)
+	runComposerCLI(t, false, "blueprints", "list")
+	// runCommand(false, "blueprints", "show", BLUEPRINT,....)
+	// runCommand(false, "blueprints", "changes", BLUEPRINT,....)
+	// runCommand(false, "blueprints", "diff", BLUEPRINT, FROM/NEWEST, TO/NEWEST/WORKSPACE)
+	// runCommand(false, "blueprints", "save", BLUEPRINT,...)
+	// runCommand(false, "blueprints", "delete", BLUEPRINT)
+	// runCommand(false, "blueprints", "depsolve", BLUEPRINT,...)
+	// runCommand(false, "blueprints", "push", BLUEPRINT.TOML)
+	// runCommand(false, "blueprints", "freeze", BLUEPRINT,...)
+	// runCommand(false, "blueprints", "freeze", "show", BLUEPRINT,...)
+	// runCommand(false, "blueprints", "freeze", "save", BLUEPRINT,...)
+	// runCommand(false, "blueprints", "tag", BLUEPRINT)
+	// runCommand(false, "blueprints", "undo", BLUEPRINT, COMMIT)
+	// runCommand(false, "blueprints", "workspace", BLUEPRINT)
+	runComposerCLI(t, false, "modules", "list")
+	runComposerCLI(t, false, "projects", "list")
+	runComposerCLI(t, false, "projects", "info", "filesystem")
+	runComposerCLI(t, false, "projects", "info", "filesystem", "kernel")
+	runComposerCLI(t, false, "sources", "list")
+	// runCommand(false, "sources", "info", "fedora")
+	// runCommand(false, "sources", "info", "fedora", "fedora-updates")
+	// runCommand(false, "sources", "add" SOURCES.TOML)
+	// runCommand(false, "sources", "change" SOURCES.TOML)
+	// runCommand(false, "sources", "delete" SOURCES.TOML)
+	runComposerCLI(t, false, "status", "show")
+
+	// Full integration tests
+	testCompose(t, "ami")
+}
+
+func testCompose(t *testing.T, outputType string) {
+	tmpdir := NewTemporaryWorkDir(t, "osbuild-tests-")
+	log.Println(tmpdir.Path)
```

Comment:
I don't think we need logging here, at least other PRs have removed logging lines when converting the syntax.

Reply:
OK, all occurrences of logging functions have been removed.